ZividOpenCV/ZDF2OpenCV/ZDF2OpenCV.cpp

```diff
@@ -0,0 +1,172 @@
+/*
+Import a ZDF point cloud and convert it to OpenCV format.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/opencv.hpp>
+
+enum class Axis
+{
+    X,
+    Y,
+    Z
+};
+
+template<Axis c>
+static float getValue(const Zivid::Point &p);
+
+template<Axis c>
+static bool minNan(const Zivid::Point a, const Zivid::Point b);
+
+template<>
+static float getValue<Axis::X>(const Zivid::Point &p)
+{
+    return p.x;
+}
+
+template<>
+static float getValue<Axis::Y>(const Zivid::Point &p)
+{
+    return p.y;
+}
+
+template<>
+static float getValue<Axis::Z>(const Zivid::Point &p)
+{
+    return p.z;
+}
+template<Axis c>
+static bool maxNan(Zivid::Point a, Zivid::Point b)
```

Comment:
I don't think it matters because (I think) if there is at least one element in the point cloud that is not a NaN it will return that element. I am talking about this: auto *maxZ = std::max_element(pointCloud.dataPtr(), pointCloud.dataPtr() + pointCloud.size(), maxNan<Axis::Z>);
Bernhard helped me with these functions and I am not entirely sure how they work.

Reply:
Ok, then it seems `max_element` is ok with `a < b` and `b < a` both being true for some elements in the collection. Since this is just for a sample, not a library function, I think we can leave it like this.

Reply:
Ok